wagtail_live/receivers.py

```diff
@@ -48,13 +50,28 @@ def is_embed(text):
 class BaseMessageReceiver:
     """Base Receiver class."""
 
-    def __init__(self, app_name, model_name):
+    @cached_property
+    def model(self):
         """
         LivePageMixin is an abstract class, so we can't make queries directly
-        We have to get the actual page which subclasses it to perform queries.
+        We have to get the actual model which subclasses it to perform queries.
         """
 
-        self.model = apps.get_model(app_name, model_name)
+        error_msg = "You won't be able to use Wagtail Live features without this setting defined."
+
+        app_name = getattr(settings, "LIVE_APP", "")
+        if not app_name:
+            raise ImproperlyConfigured(
+                "You haven't specified a live app in your settings" + error_msg,
+            )
+
+        model_name = getattr(settings, "LIVE_PAGE_MODEL", "")
+        if not model_name:
+            raise ImproperlyConfigured(
+                "You haven't specified a live page model in your settings" + error_msg,
+            )
```

Comment:
Should we check here as well that the page extends LivePageMixin? Because that is a requirement for the LIVE_PAGE_MODEL right?

Reply:
I added the check. It is sound to do this check imo.